### PR TITLE
Nuke: Allow for knob values to be validated against multiple values.

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
Knob values can now be validated against multiple values, so you can allow write nodes to be `exr` and `png`, or `16-bit` and `32-bit`.

## Testing notes:
1. Add `datatype` of `32 bit float` in `project_settings/nuke/imageio/nodes/requiredNodes/0/knobs/1/type`.
2. Create `render` instance in Nuke and modify the write node inside the output node, to use a different datatype.
3. Publish.

### CODE REVIEW

**Comment by moonyuet** [comment]
tried the validation with the reference targets , it works with `32 bit float` and some custom knob values.
image

**Review by Minkiu — APPROVED** [review]
Haven't tested, but code looks good.

File: openpype/hosts/nuke/plugins/publish/validate_write_nodes.py [hunk 1]
```diff
@@ -105,28 +112,33 @@ def process(self, instance):
                 )
 
             key = knob_data["name"]
-            value = knob_data["value"]
+            values = values_by_name[key]
```
Comment (Minkiu) [inline openpype/hosts/nuke/plugins/publish/validate_write_nodes.py:115]: I'd probably change these to more meaningful names.

Comment (tokejepsen) [reply openpype/hosts/nuke/plugins/publish/validate_write_nodes.py:115]: Got any suggestions?

**Review by jakubjezek001 — APPROVED** [review]
CLG!

**Pull request merged**